govCMS/ui-kit-base-theme
# [horizontal-layout] Added custom layout for horizontal list.

Added custom layout. 

To test:

1. Select UI KIT horizontal list with image layout in your display mode
2. Create a view to display the content type using the display mode
3. Set View format to be HTML list
4. Add `list-horizontal` in List class
5. Check the view page following the UI KIT horizontal list style
